Detectors/ITSMFT/ITS/simulation/src/Detector.cxx

```diff
@@ -917,8 +917,8 @@ void Detector::constructDetectorGeometry()
 
     LOG(DEBUG1) << "mBuildLevel: " << mBuildLevel[j] << FairLogger::endl;
 
-    if (mStaveThickness[j] != 0) {
-      mGeometry[j]->setStaveThick(mStaveThickness[j]);
+    if (mChipThickness[j] != 0) {
+      mGeometry[j]->setChipThick(mChipThickness[j]);
```

Comment:
Indeed, then it is probably not necessary, I would even go further and remove the v1 class from the repository